VideoLocker/src/androidTest/java/org/edx/mobile/test/http/ApiTests.java

```diff
@@ -0,0 +1,180 @@
+package org.edx.mobile.test.http;
+
+import com.google.gson.Gson;
+
+import org.edx.mobile.http.Api;
+import org.edx.mobile.model.api.AnnouncementsModel;
+import org.edx.mobile.model.api.AuthResponse;
+import org.edx.mobile.model.api.EnrolledCoursesResponse;
+import org.edx.mobile.model.api.HandoutModel;
+import org.edx.mobile.model.api.ProfileModel;
+import org.edx.mobile.model.api.ResetPasswordResponse;
+import org.edx.mobile.model.api.SectionEntry;
+import org.edx.mobile.model.api.SyncLastAccessedSubsectionResponse;
+import org.edx.mobile.model.api.VideoResponseModel;
+import org.edx.mobile.module.registration.model.RegistrationDescription;
+import org.edx.mobile.test.BaseTestCase;
+import org.edx.mobile.util.Environment;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+
+/**
+ * This class contains unit tests for API calls to server.
+ *
+ * if we run it in the CI of github, we can not provide the credential to
+ * make the service call.
+ * unless we find a way to handle it,  we will disable all the testing agaist
+ * real webservice right now
+ * 
+ */
+public class ApiTests extends BaseTestCase {
+
+    private Api api;
+
+    @Override
+    protected void setUp() throws Exception {
+        super.setUp();
+
+        Environment env = new Environment();
+        env.setupEnvironment(getInstrumentation().getTargetContext());
+
+        api = new Api(getInstrumentation().getTargetContext());
+    }
+    
+//    public void testSyncLastSubsection() throws Exception {
```

Comment:
Is there a `skipTest` equivalent you can use to skip these tests for now, instead of commenting them out?